srcpkgs/j/INSTALL

```diff
@@ -0,0 +1,10 @@
+case "${ACTION}" in
```

Comment:
per the comments some days ago in IRC, this is unacceptable.  You should wrap the launch script with something that determines if the local instruction set has AVX and then PRELOAD's the correct library.